keylime/tenant.py

```diff
@@ -219,6 +229,34 @@ def init_add(self, args):
             # Process allowlists
             self.allowlist = ima.process_allowlists(al_data, excl_data)
 
+        # Read command-line path string TPM2 event log template
+        if "mb_refstate" in args and args["mb_refstate"] is not None:
+            _policy_pcrs = list(self.tpm_policy.keys())
+            _policy_pcrs.remove('mask')
+            _protected_pcrs = config.MEASUREDBOOT_PCRS
+
+            for _pcr in _policy_pcrs :
+                if int(_pcr) in _protected_pcrs :
+                    logger.error(f"WARNING: PCR {_pcr} is specified in \"tpm_policy\", but will in fact be used by TPM2 measured boot. Please remove it from policy")
+                    exit()
```

Comment:
This would be an intersection of two lists of `int`s, no?
https://www.kite.com/python/answers/how-to-find-the-intersection-of-two-lists-in-python